src/auth.py

```diff
@@ -5,84 +5,95 @@
 from urllib.parse import urljoin
 
 import requests
+from beartype import beartype
 
 from .config import get_value as get_config
 from .exceptions import DeepOriginException
 
 __all__ = [
-    "get_do_api_tokens",
-    "cache_do_api_tokens",
-    "read_cached_do_api_tokens",
-    "remove_cached_do_api_tokens",
-    "sign_into_do_platform",
-    "refresh_access_to_do_platform",
+    "get_tokens",
+    "cache_tokens",
+    "read_cached_tokens",
+    "remove_cached_tokens",
+    "authenticate",
+    "refresh_tokens",
 ]
```

Comment:
i found these names highly confusing and i was never sure what actions these functions performed. renamed these functions to reflect what they actually do

Reply:
Sounds good.